### DISCUSSION TITLE
Erro redirecionamento de DNS

### DISCUSSION THREAD

**Discussion by LeonardoGattermann** [vercel/community#7232] [Help]
### Summary

Estou enfrentando alguns problemas referentes a DNS, tenho um domínio da hostiger, aonde de la estou apontando para os servidores do vercel! 
Aonde eu ja validei isso com algumas LP diretamente pelo dominio e subdominios que eu havia criado pelo vercel!

Porem ao tentar criar um subdominio "api" e aponta-lo para o ip de minha maquina, ele não "atualiza" o dns , e ja se passou mais de 24h, alguem ja passou por isso? sabe oque pode estar acontecendo ?
Testei desde o primeiro momento que fiz a troca em sites como "dnschecker.org" para verificar o endereço final, mas continua igual..

### Example

dnschecker.org

### Steps to Reproduce

a

**Comment by amyegan** [comment]
Hey @LeonardoGattermann. We have some domain troubleshooting tips to help with this: [vercel.com/docs/projects/domains/troubleshooting](https://vercel.com/docs/projects/domains/troubleshooting)

If you let me know the domain I can also look into it for you.

**Reply by LeonardoGattermann** [reply]
Bom, acredito que o problema seja o NameServer, pois por algum motivo alterou (usando o dns checker) da um resultado diferente do que foi configurado na Hostinger, tentei "atualizar" na hostinger e ver o que acontece!

Dominio: devoper.dev

**Reply by amyegan** [reply]
Thanks for sharing that additional information. Since the nameservers you have set are not being respected, you should contact Hostinger for help. They'll be able to help you investigate the cause of this DNS issue.

https://www.hostinger.com/contacts

[a comment by an automated account is not shown]
